### DISCUSSION TITLE
Computer version?

### DISCUSSION THREAD

**Discussion by lilyrosepunk** [arianneorpilla/jidoujisho#359] [Ideas]
Hi, I was just wondering if the Jidoujisho could be made to be compatible with computers for example windows and Mac. If it could that would be greatly appreciated!!!! Thankyou!!!! Also I love it, the mobile app works great!!! Thankyou!!!!

**Comment by brishtibheja** [comment]
just in case
![Screenshot_2024-02-23-13-29-00-70_40deb401b9ffe8e1df2f1cc5ba480b12](https://github.com/lrorpilla/jidoujisho/assets/136738526/a6d175ac-5225-4925-8479-a383e2e1110e)

**Reply by lilyrosepunk** [reply]
Ahh but I cant open apk files with my computer and with emulators I cant download dictionaries because the emulators dont use my computers storage. Thank you though.

**Reply by xrishox** [reply]
emulators do work. i've used it with both windows subsystem for android and android studio(and waydroid). it also works with waydroid on linux. it isn't the best user experience though

**Reply by ashirogisensei** [reply]
if you have a samsung phone, you can download samsung dex onto your windows pc and install it. then you just plug your phone into your windows pc and you can use jidoujisho on your computer. That is assuming that you've set up jidoujisho up on your samsung phone
